ignition-server/controllers/machineconfigserver_ignitionprovider.go

```diff
@@ -183,6 +184,15 @@ func (p *MCSIgnitionProvider) GetPayload(ctx context.Context, releaseImage strin
 const mcoBootstrapScript = `#!/bin/bash
 mkdir -p /mcc-manifests/bootstrap/manifests
 mkdir -p /mcc-manifests/manifests
+{{ if .isAzure }}
+cat <<EOF >/tmp/cloud.conf.configmap.yaml
```

Comment:
why is this needed?

Reply:
Because the kubelet panics if its started with the Azure cloud provider but no cloud config

Reply:
Is that only for azure or also for AWS?
Is there an issue open against kubelet azure provider code to discuss?
Can we unify code and do this unconditionally so no need for the ignition-server to know the platform?